packages/openapi/src/openapi-request-builder.ts

```diff
@@ -39,11 +40,13 @@ export class OpenApiRequestBuilder<ResponseT = any> {
    * @param method - HTTP method of the request to be built.
    * @param pathPattern - Path for the request containing path parameter references as in the OpenAPI specification.
    * @param parameters - Query parameters and or body to pass to the request.
+   * @param basePath - The path to be prefixed to the pathPattern.
```

Comment:
```suggestion
   * @param basePath - The path to be prefixed.
```

Reply:
Similar to my previous comment but not that problematic this time. But still always point out what it is by saying the <name> [method | parameter | function | ...]

Reply:
Maybe `to the API path parameter` or just the API path?

Reply:
I like API path parameter, will use that instead.